README.rst

```diff
@@ -182,7 +182,7 @@ RLlib Quick Start
 More Information
 ----------------
 
-- `Documentation`_
+- `Documentation`_, in particular `Building Ray and Contributing to Ray`_
```

Comment:
Not sure it makes sense to emphasize "building and contributing to Ray" here. The vast majority of users will be looking for the API.

Reply:
It doesn't hurt to hint at getting people to contribute 😄

Reply:
Maybe it's more common in the JVM world, where it's not uncommon for people to want to try building the library themselves, even if they mostly intend to just use prebuilt libraries. It's a sanity check of code and project quality if building is straightforward and reliable. Not always of course. In our case, it's too complicated for the details to be summarized here, since there is a mixture of C++ and Python.

Reply:
I'd rather put this in the "Getting Involved" section.